codex_of_the_damned/templates/archetypes/bibliodeque/anson-grooming.html

```diff
@@ -15,29 +15,69 @@ <h1><img src="{{ url_for('static', filename='img/icons/clan-toreador.svg') }}">
 
 {% block archetype %}
 {% trans trimmed
-
+	anson=card("Anson"),
+	tatiana=card("Tatiana Romanov"),
+	voter_cap=card("Voter Captivation"),
+	krc=card("Kine Resources Contested"),
+	tgb=card("Toreador Grand Ball"),
+	grooming=card("Grooming the Protégé"),
+	parity_shift=card("Parity Shift"),
+	toreador_justicar=card("Toreador Justicar"),
+	elder_library=card("Elder Library"),
+	bloat=link("/strategy/bloat", _("bloat")),
+	vote=link("/strategy/archetypes", _("Vote"), _anchor="vote"),
+	politics=link("/strategy/fundamentals", _("Political Actions"), _anchor="politics"),
+	inaki_jimenez_deck=link("/deck-search", _("Iñaki Jimenez's deck"), twda_id="2k9fsbarakaldolate")
 %}
 <h2>Highlights</h2>
 <p>
-
+	This is a <em>breed & boon</em> deck starring {{ anson }} and using {{ PRE }} as the
+	discipline to push votes. The purpose is to have him out as fast as possible
+	to use {{ politics }} to oust the prey.
 </p>
 
 <h2>Tips & Tricks</h2>
 <p>
-
+	Using {{ grooming }} cost quite an amount of blood and it is necessary to
+	replenish vampires from time to time. {{ voter_cap }} can be used to do this
+	instead of hunting as much as possible. This solution does not cost more actions
+	and it will be easier to maximise damage done to the prey.
+</p>
+<p>
+	{{ krc }} is the best action to deal massive damage. It is possible to
+	negociate with the great predator how to assign damage if their votes are necessary.
+	It is important to still deal more damage to the prey than to the predator because it
+	will only help the great predator do their point and become the predator more quickly
```

Comment:
```suggestion
	It is important to still deal more damage to the prey than to the predator. Otherwise, it
	will only help the great predator oust their prey and become the predator more quickly.
```

Reply:
👍